learn/tanssi/technical-features.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Block Production Services
 description: Tanssi provides block production services assigning block producers to the appchains, requiring minimal changes to the code for appchains to be deployed.
+icon: material-application-cog-outline
```

Comment:
Maybe trying this one would help align with the other icons better: `octicons-gear-24`